.github/workflows/build.yaml

```diff
@@ -0,0 +1,34 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 2
+      matrix:
+        java: [8, 11, 15]
```

Comment:
May I ask why you now want to run on Java 8 and the NON LTS 15, when you ran every build before on Java 11

Reply:
Good question. I wanted to include all LTS releases and the latest release. So that's why 15 is in the list.

Reply:
Java 8 is the default on Android platforms nowadays, while Java 15 is the latest stable version available. I think it makes sense.

Reply:
fair enough! thank you for the clarification :wink:

Reply:
Note that the apiclient works with Java applications too, it's not specifically meant for Android apps.